coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -42,20 +43,53 @@ export function verifyPrescriptionBundle(bundle: fhir.Bundle): Array<errors.Vali
     'extension("https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-MedicationRepeatInformation")'
   ]
   const medicationRequests = getMedicationRequests(bundle)
-  return fhirPaths
+  const identicalValueErrors = fhirPaths
     .map((fhirPath) => verifyIdenticalForAllMedicationRequests(bundle, medicationRequests, fhirPath))
     .filter(Boolean)
+  const repeatDispensingErrors = verifyRepeatDispensingPrescription(medicationRequests)
+  return [...identicalValueErrors, ...repeatDispensingErrors]
 }
 
-function verifyCancellationBundle(bundle: fhir.Bundle): Array<errors.ValidationError> {
-  //TODO - implement cancellation-specific validation
-  bundle
+export function verifyRepeatDispensingPrescription(
+  medicationRequests: Array<fhir.MedicationRequest>
+): Array<errors.ValidationError> {
+  if (getCourseOfTherapyTypeCode(medicationRequests) !== CourseOfTherapyTypeCode.CONTINUOUS_REPEAT_DISPENSING) {
+    return []
```

Comment:
should this return an error?

Reply:
No, this just means the prescription is not a repeat dispensing one, so none of the validation applies

Reply:
I think based on this question being asked it's probably clearer if the check for the course of therapy type is outside of this method and this method is only called for repeat dispensing medication requests to avoid confusion. It's not obvious from the method name that this can be run against acute prescriptions without raising errors

Reply:
Sorted